packages/main/src/Tokenizer.js

```diff
@@ -256,9 +260,15 @@ class Tokenizer extends UI5Element {
 
 	_handleItemNavigation(event, tokens) {
```

Comment:
Open the test page, focus a sample containing tokens at least two tokens. With arrow left go to a token different from the last one. Then try to go to the token on the right of the focused one, using Arrow Down key -> does not work. If you try with Control + Arrow Down it works. Moreover, after an interaction with Control + Arrow Down, the Arrow Down navigation among the tokens works as well.